app/eventyay/common/templates/common/base.html

```diff
@@ -107,15 +107,15 @@ <h1>
                                     </summary>
                                     <div id="locale-dropdown" class="dropdown-content dropdown-content-s{% if rtl %}e{% else %}w{% endif %}">
                                         {% for l, name in request.event.named_locales %}
-                                            <a href="{% url "cfp:locale.set" event=request.event.slug %}?locale={{ l }}&next={{ request.path }}%3F{{ request.META.QUERY_STRING|urlencode }}"
+                                            <a href="{% url "cfp:locale.set" organizer=request.event.organizer.slug event=request.event.slug %}?locale={{ l }}&next={{ request.path }}%3F{{ request.META.QUERY_STRING|urlencode }}"
```

Comment:
It is error-prone to build URL in the template. Please move the code to Python file, applying [`urlencode`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode), because some characters are illegal in URL.